### ISSUE TITLE
Text chat or Text to speech or Built in Mute Helper

### ISSUE THREAD

**Issue by Pinkyyyy** [Yellow-Dog-Man/Resonite-Issues#50]
### Is your feature request related to a problem? Please describe.

So I don't have a mic setup, and sometimes maybe people don't feel like talking, a native way for people without mics to talk to other people would be great, and in my opinion necessary. It is extremely cute to just be able to nod your head yes or no, but it's a limitation. 

### Describe the solution you'd like

Had a pleasure of being given a mute helper by the worlds host which made the whole experience a lot more natural feeling, even more when they gave me an add-on for text to speech. A solution similar to this one would work really well, with options for people to mute peoples text chat, host to mute someone in text, or give them permission to use text to speech etc. etc.

You could also have a text cloud next to the person, to see that they are typing, and then what was typed

### Describe alternatives you've considered

Currently there doesn't seem to be a way to communicate natively in the game unless you have a microphone, the only other solution i saw was someone handing me a text box to type into, which was clunkier than the Mute Helper because i have to carry it around with me, and the Mute Helper just attaches to my avatar. It also resize text nicely, etc. etc. Plus the way Mute Helper handles text, by making it float next to your character, is a nice way to handle text chat, as it's not just a text box on the screen that gets cluttered and you don't know if the person typing there is close to you or not.

### Additional Context

Apologies if this issue is formatted wrong, first time doing this.

**Comment by InconsolableCellist** [comment]
:+1: Super important for deaf and hard-of-hearing people as well.

**Comment by Pinkyyyy** [comment]
Definitely, a speech to text option would also be great, i think apex legends has this ability, not quite sure how well it works though.

**Comment by NukiRaccoon** [comment]
Popping in to Second this request. 

Was talking to BrainInnaJar, who doesn't have a microphone, and she pointed out that she has an ongoing issue of loading into worlds without her Avatar and not being able to spawn an item to either put on the Avatar or put mute helper on the default avatar, and that resonite is the only social platform she has ever used that doesn't have a built-in text chat feature.

Keeping that in mind, I am beginning to realize just how complicated Resonite quickly becomes with the current way that Mute Helper is set up: new users who don't have voice available at that moment are basically told to navigate the inventory, which they have never navigated before, to go down several levels in the public folder hierarchy, spawn out an item for the first time, then install it to their Avatar by either pulling it into their face or using the radial menu they just learned about... And they have to do all that before they have any means of communication.

This is bad for a new user experience, especially when trying to convince them that Resonite is not that complicated and that everything is not going to be like that.

A simple implementation of this would be to put in some way to check if a user's avatar already has mute helper, and if not, add it to their user root.  Then, instead of having to show them how to navigate pu, all we have to do is tell them to press the 'H' key. 

Please add me and BrainInnaJar as requesters.

**Comment by Dusty-Sprinkles** [comment]
I am once again requesting that someone with the ability to edit github issues updates this issue (title & description) to reflect creating an official mute helper interface and having it be something that spawns with users because the tts is in a different issue
